frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -213,6 +213,13 @@ function FileManagerMenu:setUpdateItemTable()
                                 purgeDir(cachedir)
                             end
                             lfs.mkdir(cachedir)
+                            -- Clear Cache object, so it doesn't serialize
+                            -- anything old on exit
+                            local Cache = require("cache")
+                            Cache:clear()
```

Comment:
There's already this prompt `text = _("Clear cache/ and cr3cache/ ?")`.
Should I add "Please restart koreader now." to this message or in an InfoMessage after it's done ?
Or do you have another idea for "forcing user to restart" ? :)

Also, to avoid this crash on exit (sorting according to file timestamps for files in cached but no more on disk, so with lfs.attributes("access") = nil):
```
./luajit: frontend/cache.lua:159: attempt to compare two nil values
stack traceback:
        frontend/cache.lua:159: in function <frontend/cache.lua:159>
        [C]: in function 'sort'
        frontend/cache.lua:159: in function 'serialize'
```
I think it's still needed to just do `Cache.cached = {}` .
Would that be fine ?